### PR TITLE
component refactoring to run jquery code outside angular

### PR DESCRIPTION
**Pull request by pwpatton**
- soho-busy-indicator.directive.ts
- soho-button.component.ts
- soho-context-menu.directive.ts
- soho-expandablearea.component.ts
- soho-popupmenu.component.ts
- soho-tabs.component.ts
- soho-toolbar.component.ts

### CODE REVIEW

File: DEVELOPER.md [hunk 1]
```diff
@@ -132,40 +132,56 @@ private widget: SohoWidgetStatic;
...
 
 ```typescript
 ngAfterViewInit() {
-  this.jQueryElement = jQuery(this.element.nativeElement);
-  this.jQueryElement.widget(this.options);
-  this.widet = this.jQueryElement.data('widget');
-  this.jQueryElement
-    .on('built', (e: JQueryObjectEvent, widgetId: string) => this.built.next({widgetId}));
+  this.ngZone.runOutsideAngular(() => {
+    // initialize the jqueyr component outside of angular.
```
Comment (tmcconechy) [inline DEVELOPER.md]: jquery

File: src/app/tabs/test-tabs-basic.demo.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+@Component({
+  selector: 'test-tabs-basic', // tslint:disable-line
+  templateUrl: './test-tabs-basic.demo.html',
+})
+export class TestTabsBasicComponent implements AfterViewChecked {
+  private viewCheckCount = 0;
+  public tabActivatedCount = 0;
+
+  ngAfterViewChecked() {
+    console.log('TestTabsBasicComponent.ngAfterViewChecked called ' + (++this.viewCheckCount) + ' times');
```
Comment (tmcconechy) [inline src/app/tabs/test-tabs-basic.demo.ts]: Do we need to keep these logs in permanent

Comment (pwpatton) [reply src/app/tabs/test-tabs-basic.demo.ts]: Probably a toast message would be better.

Comment (pwpatton) [reply src/app/tabs/test-tabs-basic.demo.ts]: Yes, all tests run and samples tested.

**Pull request merged**